modules/runner/src/integrationTest/java/org/apache/ignite/internal/configuration/storage/ItRebalanceDistributedTest.java

```diff
@@ -962,6 +1123,22 @@ TablePartitionId getTablePartitionId(String tableName, int partitionId) {
 
             return new TablePartitionId(internalTable.tableId(), partitionId);
         }
+
+        public String getName() {
```

Comment:
Node is a private class only for this test class. I would prefer to just make the fields public, intead of spam test itself with these getters

Reply:
Fixed.